shims/common/src/main/java/org/apache/hadoop/hive/thrift/HadoopThriftAuthBridge.java

```diff
@@ -286,8 +289,13 @@ static String encodeIdentifier(byte[] identifier) {
       HIVESERVER2, METASTORE
     };
 
+    public static final String HIVE_SERVER2_KERBEROS_CUSTOM_AUTH_CLASS =
```

Comment:
Since, HadoopThriftAuthBridge class does not have HiveConf. Therefore, even I set the HiveConf as global variable, HadoopThriftAuthBridge cannot read the value. That's why I defined two different places.